docs/source/contributing.rst

```diff
@@ -0,0 +1,121 @@
+Contributing
+============
+
+poliastro is a community project, hence all contributions are more than
+welcome!
+
+Bug reporting
+-------------
+
+Not only things break all the time, but also different people have different
+use cases for the project. If you find anything that doesn't work as expected
+or have suggestions, please refer to the `issue tracker`_ on GitHub.
+
+.. _`issue tracker`: https://github.com/poliastro/poliastro/issues
+
+Documentation
+-------------
+
+Documentation can always be improved and made easier to understand for
+newcomers. The docs are stored in text files under the `docs/source`
+directory, so if you think anything can be improved there please edit the
+files and proceed in the same way as with `code writing`_.
+
+The Python classes and methods also feature inline docs: if you detect
+any inconsistency or opportunity for improvement, you can edit those too.
+
+Besides, the `wiki`_ is open for everybody to edit, so feel free to add
+new content.
+
+To build the docs, you must first create a development environment (see
+below) and then in the ``docs/`` directory run::
+
+    $ cd docs
+    $ make html
+
+After this, the new docs will be inside ``build/html``. You can open
+them by running an HTTP server::
+
+    $ cd build/html
+    $ python -m http.server
+    Serving HTTP on 0.0.0.0 port 8000 ...
+
+And point your browser to http://0.0.0.0:8000.
+
+Code writing
+------------
+
+.. image:: https://img.shields.io/waffle/label/poliastro/poliastro/1%20-%20Ready.svg?style=flat-square
+   :target: https://waffle.io/poliastro/poliastro
+   :alt: 'Stories in Ready'
+
+Code contributions are welcome! If you are looking for a place to start,
+help us fixing bugs in poliastro and check out the `"easy" tag`_. Those
+should be easier to fix than the others and require less knowledge about the
+library.
+
+.. _`"easy" tag`: https://github.com/poliastro/poliastro/issues?q=is%3Aissue+is%3Aopen+label%3Aeasy
+
+If you are hesitant on what IDE or editor to use, just choose one that
+you find comfortable and stick to it while you are learning. People have
+strong opinions on which editor is better so I recommend you to ignore
+the crowd for the time being - again, choose one that you like :)
+
+If you ask me for a recommendation, I would suggest PyCharm (complete
+IDE, free and gratis, RAM-hungry) or vim (powerful editor, very lightweight,
+steep learning curve). Other people use Spyder, emacs, gedit, Notepad++,
+Sublime, Atom...
+
+You will also need to understand how git works. git is a decentralized
+version control system that preserves the history of the software, helps
+tracking changes and allows for multiple versions of the code to exist
+at the same time. If you are new to git and version control, I recommend
+following `the Try Git tutorial`_.
+
+.. _`the Try Git tutorial`: https://try.github.io/
```

Comment:
I think this tutorial is also very good and worth adding:

https://www.asmeurer.com/git-workflow/

Reply:
Added!